docs/references/config/config.toml.md

```diff
@@ -0,0 +1,1749 @@
+a CometBFT perspective. However, the GoLevelDB library has not been maintained in the past two years.
+Choose your poison.
+
+### db_dir
+The directory path where the database is stored.
+```toml
+db_dir = "data"
+```
+
+| Value type          | string                                                      |
+|:--------------------|:------------------------------------------------------------|
+| **Possible values** | relative directory path, appended to `$CMTHOME` |
+|                     | absolute directory path                         |
+
+The default relative path translates to `$CMTHOME/data`. In case `$CMTHOME` is unset, it defaults to
+`$HOME/.cometbft/data`.
+
+### log_level
+A comma-separated list of `module:level` pairs that describe the log level of each module. Alternatively, a single word
+can be set which will apply that log level to all modules.
+```toml
+log_level = "info"
+```
+
+| Value type     | string          |                                        |
+|:---------------|:----------------|----------------------------------------|
+| **Modules**    | `"main"`        | CometBFT main application logs         |
+|                | `"consensus"`   | consensus reactor logs                 |
+|                | `"p2p"`         | p2p reactor logs                       |
+|                | `"pex"`         | Peer Exchange logs                     |
+|                | `"proxy"`       | ABCI proxy service (MultiAppConn) logs |
+|                | `"abci-client"` | ABCI client service logs               |
+|                | `"rpc-server"`  | RPC server logs                        |
+|                | `"txindex"`     | Indexer service logs                   |
+|                | `"events"`      | Events service logs                    |
+|                | `"pubsub"`      | PubSub service logs                    |
+|                | `"evidence"`    | Evidence reactor logs                  |
+|                | `"statesync"`   | StateSync reactor logs                 |
+|                | `"mempool"`     | Mempool reactor logs                   |
+|                | `"blocksync"`   | BlockSync reactor logs                 |
+|                | `"state"`       | Pruner service logs                    |
+|                | `"*"`           | All modules                            |
+| **Log levels** | `"debug"`       |                                        |
+|                | `"info"`        |                                        |
+|                | `"error"`       |                                        |
+|                | `"none"`        |                                        |
+
+At the end of a `module:level` list, a default log level can be set for modules with no level set. Use `*` instead of a
+module name to set a default log level. The default is `*:info`.
+
+Examples:
+
+Set the consensus reactor to `none` log level and the `p2p` reactor to `debug`. Everything else should be set to error:
```

Comment:
This is a bad example, although valid. Put consensus on debug and p2p as none, ehehe